src/main/kotlin/pt/up/fe/ni/website/backend/email/SimpleEmailBuilder.kt

```diff
@@ -0,0 +1,72 @@
+package pt.up.fe.ni.website.backend.email
+
+import jakarta.activation.DataSource
+import jakarta.activation.FileDataSource
+import jakarta.activation.URLDataSource
+import java.io.File
+import java.net.URL
+import org.springframework.mail.javamail.MimeMessageHelper
+
+class SimpleEmailBuilder : BaseEmailBuilder() {
+    private var text: String? = null
+    private var html: String? = null
+    private var subject: String? = null
+    private var attachments: MutableList<EmailFile> = mutableListOf()
+    private var inlines: MutableList<EmailFile> = mutableListOf()
```

Comment:
What are inlines?

Reply:
Oh, nice. Maybe we should have a comment explaining that since I'm guessing a lot of people won't know that.